modules/permutiveRtdProvider.js

```diff
@@ -121,16 +125,34 @@ function getDefaultBidderFn (bidder) {
     },
     trustx: function (bid, data, acEnabled) {
       if (acEnabled && data.ac && data.ac.length) {
-        deepSetValue(bid, 'params.keywords.p_standard', data.ac)
+        setBidderRtb('trustx', data.ac)
```

Comment:
This is the exact logic to pass data to any generic bidder, instead of whitelabeling just trustx, please make bidder name configurable do publishers can call this victim for any bidder.